spec/features/user_recievs_email_after_microhood_is_reviewed_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'User is notified via email after review of their created microhood' do
+  scenario 'user reviews microhood' do
+    ActionMailer::Base.deliveries = []
+    user = FactoryGirl.create(:user)
```

Comment:
You're doing the same setup for both tests within this scope. It would clean up your code to bring these out into let blocks within the feature block